hooks/build.sh

```diff
@@ -5,3 +5,4 @@ SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
 cd $SCRIPT_DIR/../lambda/custom
 yarn build
 yarn zip
+yarn copy-build
```

Comment:
@cdubz Turns out that the build.zip file needs to be copied to the ```.ask``` folder and that is the one that is used to upload to the lambda function.  I noticed this when I wasn't seeing the changes I was making after deploying.

Reply:
Glad you found that before I had a chance to test 😄

Will take a look at this tonight, hopefully.